db/blip_handler.go

```diff
@@ -552,9 +554,20 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, opts *sendChangesOptions
 	return (err == nil && !forceClose), err
 }
 
-func (bh *blipHandler) buildChangesRow(change *ChangeEntry, revID string) []interface{} {
+func (bh *blipHandler) buildChangesRow(change *ChangeEntry, changeVersion ChangeByVersionType) []interface{} {
 	var changeRow []interface{}
 
+	// change map should only have one entry
+	if len(changeVersion) > 1 {
+		base.AssertfCtx(bh.loggingCtx, "more changes in list than expected on change entry: %v", change.ID)
+	}
+	var rev string
+	if revID, ok := changeVersion[ChangesVersionTypeRevTreeID]; ok {
+		rev = revID
```

Comment:
Everywhere else has kind of prioritized `CV` and have a safe fallback to `RevTreeID` - this is the reverse. I know we have assertions in place to ensure only one value type, but it might still make sense to be consistent about order of precedence between the two.